testproject/Assets/StreamingAssets/empty.txt.meta

```diff
@@ -0,0 +1,7 @@
+fileFormatVersion: 2
```

Comment:
what's this `empty.txt` (and it's meta file)? :)

I guess we could just simply put `/[Aa]ssets/[Ss]treamingAssets/` into `.gitignore` file above and ignore all the stuff under that directory, what do you think?

Reply:
I'm not a big fan of ignoring unity magic folders.
I'm also not a big fan of having scripts creating project directories in the background, especially magic folders like this
If for whatever reason we need StreamingAssets for other stuff later, we'll be wondering why it doesn't appear in git and who's creating what.

Reply:
huh, after a quick research I'm convinced that it should stay — [it's a special kind of folder (doc here →)](https://docs.unity3d.com/Manual/StreamingAssets.html).